src/vs/workbench/contrib/issue/browser/issueFormService.ts

```diff
@@ -64,6 +62,15 @@ export class IssueFormService implements IIssueMainService {
 		});
 
 	}
+	reloadWithExtensionsDisabled(): Promise<void> {
+		throw new Error('Method not implemented.');
+	}
+	showConfirmCloseDialog(): Promise<void> {
+		throw new Error('Method not implemented.');
+	}
+	showClipboardDialog(): Promise<boolean> {
+		throw new Error('Method not implemented.');
+	}
```

Comment:
whats up with these?

Reply:
the web reporter hadn't implemented these yet, but since it implements the new `IssueFormService`, the naming conventions just changed

Reply:
but they're called so it would throw yes? What happens? Also, shouldn't these be the same implementation as the desktop case?

Reply:
yep good point - just added it accross from the desktop. using host service instead of native host service ofc though.

tested and it looks like it works no issues